napari/components/scale_bar.py

```diff
@@ -17,6 +21,10 @@ class ScaleBar(EventedModel):
         default color is magenta. If not colored than
         scale bar color is the opposite of the canvas
         background.
```

Comment:
```suggestion
        background or the background box.
```

To describe the new behavior added in this PR. The logic is bit long/tricky to describe, so I think this short addition is sufficient. Feel free to add more.